src/server/api.js

```diff
@@ -57,12 +63,15 @@ export const CreateGame = async (db, game, numPlayers, setupData) => {
   return gameID;
 };
 
-export const createApiServer = ({ db, games }) => {
+export const createApiServer = ({ db, games, lobbyConfig }) => {
   const app = new Koa();
-  return addApiToServer({ app, db, games });
+  return addApiToServer({ app, db, games, lobbyConfig });
 };
 
-export const addApiToServer = ({ app, db, games }) => {
+export const addApiToServer = ({ app, db, games, lobbyConfig }) => {
+  if (!lobbyConfig || !lobbyConfig.shortid) {
+    lobbyConfig = { ...lobbyConfig, shortid };
```

Comment:
If lobbyConfig is `undefined` or `null`, then wouldn't this object-spread cause an error?  I think you want something like:

```
if (!lobbyConfig) {
  lobbyConfig = {};
}

if (!lobbyConfig.shortid) {
  lobbyConfig = { ...lobbyConfig, shortid };
}
```